randovania/games/samus_returns/generator/hint_distributor.py

```diff
@@ -13,7 +13,7 @@ class MSRHintDistributor(HintDistributor):
     @override
     @property
     def default_precision_pair(self) -> PrecisionPair:
-        return PrecisionPair(HintLocationPrecision.REGION_ONLY, HintItemPrecision.DETAILED, True)
```

Comment:
since MSR used to be region-only, I suggest overriding `use_detailed_location_precision()` to return `False`